superset-embedded-sdk/src/index.ts

```diff
@@ -34,6 +37,11 @@ export type UiConfigType = {
   hideTitle?: boolean
   hideTab?: boolean
   hideChartControls?: boolean
+  filters?: {
+    [key: string]: boolean | undefined
+    visible?: boolean
+    expanded?: boolean
```

Comment:
@jileeon - considering `DASHBOARD_UI_FILTER_CONFIG_URL_PARAM_KEY` only has 2 keys namely `visible` and `expanded` and line number 115 does `DASHBOARD_UI_FILTER_CONFIG_URL_PARAM_KEY[key]=filterConfig[key]`, i don't understand why there is a need for line number 41 (`[key: string]: boolean | undefined`) in the code.

I found this code while I was trying to add some additional capability add some extra url parameters to the embedded superset. But I don't want to add that code if the functionality already exist.

Reply:
Hi @jayakrishnankk, Sorry for the late reply. 

these code you wrote are just for complying with Typescript lint.
I think there are no need to keep them If they should be changed.

Thanks.